source/settings/plugin/advanced/obox.rst

```diff
@@ -110,6 +110,18 @@ Overrides the obox username in storage. Normally the username is taken from the
 
 Path to communicate with metacache process. Shouldn't be changed normally.
 
+.. _plugin-obox-setting_metacache_userdb:
+
+``metacache_userdb``
+--------------------
+
+- Default: ``metacache/metacache-users.db``
+- Values: :ref:`string`
+
+Path to a database which metacache process periodically writes to.
+This database is read by metacache at startup to get the latest state.
+Relative to :ref:`setting-state_dir`.
+Shouldn't be changed normally.
```

Comment:
Hmm, maybe these sentences should be made more complete. A verb would help.

Reply:
The last two? Changed them.